thrift/lib/cocoa/src/TBaseStruct.m

```diff
@@ -35,9 +35,6 @@ - (instancetype) init
     return self;
 }
 
-/**
```

Comment:
These comments are deleted?

Reply:
They are already in the header for this class. I'm fine putting them back, but in theory it's comments duplication. In Xcode if would like to look at the documentation for the API you can hit Option+Click. For this overlay it will use the header to gather the information. I will put them back though as requested.